ferrocene/doc/user-manual/src/cli.rst

```diff
@@ -166,8 +166,9 @@ Command-Line Interface
 
       .. caution::
 
-         Only the arguments listed in section Compilation targets are within
-         the scope of the Ferrocene |iso_ref| qualification.
+         This code generation option is outside the scope of the Ferrocene
```

Comment:
This is not correct, and the old text should stay: this means, you *can* use `-C link-arg`, but only with the values we describe in each target's documentation.

Besides, we should remove |iso_ref| from all of these cautions.

Reply:
This needs to be applied to the other similar changes too, it's not resolved.